components/Card.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+const CardContainer = styled.div`
```

Comment:
You should go with a list item here, so that in your app structure there are list items wrapped by an unordered list
(this leads to more semantic html)

```suggestion
const CardContainer = styled.li`
```